### DISCUSSION TITLE
Is there a way to get 8x8 tiles instead of 16x16

### DISCUSSION THREAD

**Discussion by erlendnagel** [paulscottrobson/neo6502-firmware#299] [Q&A] [answered]
The the two-color and 5x7 of user defined characters is a bit limiting and the 16x16 tiles are too big for what I want to do (pacman). Is there some middle way like an 8x8 tile that could be used?

**Comment by paulscottrobson** [comment, marked as answer]
Not at present. About the best I could do would be a hack to draw an 16x16 tilemap as an 8x8 tile. Be rather messy.

**Reply by erlendnagel** [reply]
Ok. I'll try for now to just take 8 pixel offsets in drawing overlapping tiles. The basic pacman background is static anyway.
